react/Modal/index.jsx

```diff
@@ -24,10 +26,10 @@ const ModalTitle = ({ children, className }) =>
 
 const ModalCross = ({ onClick, className }) => (
   <button
-    className={classNames(styles['coz-btn'], styles['coz-btn--close'], styles['coz-btn-modal-close'], className)}
+    className={classNames(styles['c-btn'], styles['c-btn--close'], styles['coz-btn-modal-close'], className)}
     onClick={onClick}
     >
-    <span className={styles['coz-hidden']}></span>
+    <Icon icon="cross" width="24" height="24" color={palette["coolGrey"]} />
```

Comment:
Do you mind using single quotes for consistency here?

Reply:
right… sorry. :(